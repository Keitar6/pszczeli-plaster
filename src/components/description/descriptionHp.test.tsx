import ComponentsRenderer from "react-test-renderer";
import { DescriptionHP } from "./descriptionHP.component";

describe("DescriptionHP", () => {
  test("DescriptionHPn", () => {
    const component = ComponentsRenderer.create(<DescriptionHP />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
