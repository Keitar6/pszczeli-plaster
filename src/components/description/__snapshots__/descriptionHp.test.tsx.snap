// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DescriptionHP DescriptionHPn 1`] = `
<div
  className="sc-iBkjds sc-kDDrLX gftbFx bHWBi"
>
  <h3
    className="sc-dkzDqf sc-papXJ exRqZr dWxHTb"
  >
    O NAS
  </h3>
  <div
    className="sc-jqUVSM heUhMV"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Optio similique laborum excepturi ducimus praesentium, sapiente cumque ipsum perspiciatis quod eos illum aliquam illo a, rem, recusandae veritatis quibusdam in tempora? Lorem ipsum dolor sit amet consectetur adipisicing elit. Optio similique laborum excepturi ducimus praesentium, sapiente cumque ipsum perspiciatis quod eos illum aliquam illo a, rem, recusandae veritatis quibusdam in tempora?
  </div>
</div>
`;
